manifest.json

```diff
@@ -3213,6 +3223,16 @@
                 }
             },
             "versions": {
+                "1.1.0": {
+                    "releaseUrl": "https://github.com/rassi0429/NeosPasteTweak/releases/tag/v1.1.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/rassi0429/NeosPasteTweak/releases/download/v1.1.0/NeosPasteTweak.dll",
```

Comment:
Contains huge structs that are spawned with hardcoded IDs, can't say I understand them so can't really approve of it...

Reply:
Yeah it has 2 embedded objects it recreates from an exported json of the neos objects and replaces the text. Ideally it'd remake the object directly to avoid the big chunk of hard to read stuff and all the IDs or otherwise be setup with a neosdb object link (although more fragile)

Reply:
The giant strings are funky, but tbh a valid way to do this. Only funny business is it's putting the object into userspace for some actions (if I'm reading this right) but at the end of the day this mod is no more dangerous than adding a facet to your dash, so I don't see a reason to sit on the review.